showcase/app/styles/mock-components/app.scss

```diff
@@ -18,6 +18,8 @@
   margin-bottom: 12px;
 }
 
-.mock-app-main-generic-advanced-table {
+.mock-app-main-generic-advanced-table-wrapper {
+  display: grid;
```

Comment:
testing locally, it looks like adding display grid is what makes the shadow appear correctly on initial page load.

Reply:
Nice!

Reply:
@shleewhite is this override still needed? or you changed things on the AdvancedTable side and now it's not necessary anymore?

Reply:
It is still needed, it has not been updated in the component yet. Side note: when I was converting Explorer to Advanced Table, adding `display: grid` made the grid too tall sometimes and there would be a big gap between the thead and tbody. I ended up adding `max-height: max-content` to resolve the issue.

Reply:
> It is still needed, it has not been updated in the component yet

@shleewhite OK, so when it's updated we will have to remember to remove this extra wrapper and class. Do you have a PR or Jira ticket I can reference? so I can add a comment to avoid we forget